### PR TITLE
Feature/civ 3206

### PR DESCRIPTION
**Pull request by MMNycz**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-3206

### Change description ###

Add information to general application that main case is SPEC/UNSPEC

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java [hunk 1]
```diff
@@ -105,6 +105,7 @@ public class CaseData extends CaseDataParent implements MappableObject {
     private final GAStatementOfTruth generalAppStatementOfTruth;
     private final GAHearingDetails generalAppHearingDetails;
     private final GASolicitorDetailsGAspec generalAppApplnSolicitor;
+    private final String generalAppSuperClaimType;
```
Comment (smitatalawadekar-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java:108]: why do we need this field? It is neither being set any value nor being used anywhere?

File: src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GASuperClaimType.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.civil.model.genapplication;
```
Comment (smitatalawadekar-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GASuperClaimType.java:1]: I do not understand the requirement of this class... where is it used?

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java (hunk 1 above)
Comment (MMNycz) [reply src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java]: After talking to Smita - I deleted these values as they were not needed under CaseData. Tested again works approx.

File: src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GASuperClaimType.java (hunk 2 above)
Comment (MMNycz) [reply src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GASuperClaimType.java]: After talking to Smita - I deleted these values as they were not used and forgot to delete after changing the code. Tested again works approx.

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

**Pull request merged**